storybook/src/pages/amsterdam/ProductPage/ProductPage.tsx

```diff
@@ -0,0 +1,278 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import {
+  Accordion,
+  Breadcrumb,
+  Button,
+  Grid,
+  Heading,
+  Link,
+  LinkList,
+  OrderedList,
+  Paragraph,
+  Spotlight,
+  TableOfContents,
+  UnorderedList,
+} from '@amsterdam/design-system-react'
+import './product-page.css'
+
+export const ProductPage = () => (
+  <>
+    <Grid paddingTop="small">
+      <Grid.Cell span={{ narrow: 4, medium: 5, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 3 }}>
+        <Breadcrumb>
+          <Breadcrumb.Link>Home</Breadcrumb.Link>
+        </Breadcrumb>
+      </Grid.Cell>
+    </Grid>
```

Comment:
Giving the Breadcrumb its own Grid is necessary to align the headings of the page and of the table of contents.

On a narrow screen, this sets the breadcrumb before the Table of Contents heading instead of the Page Heading. Is that acceptable?

If not, our Grid would need more reordering features.

Reply:
We give header it's own grid padding, I see the breadcrumbs as the next layer of navigation. We could contemplate doing the padding-inline thing. Makes it less tedious and will always align properly.